### ISSUE TITLE
Not working in ver. 2.4.2-p1

### ISSUE THREAD

**Issue by newalway** [prakash282/OrderAttachment#22]
I install and not appearing tab but database has been created.

**Comment by junnhouse** [comment]
I have same issue

**Comment by newalway** [comment]
I solved this, because css display none . I override css to display block will show

**Comment by junnhouse** [comment]
you mean admin order tab?

**Comment by junnhouse** [comment]
I found it with class no-display hidden . How did you override this? Thanks in advance.

**Comment by Masumisigma** [comment]
Hi, I installed the module in magento2.3.2 but the Sp – Order Attachments menu is not coming.Any idea?

**Comment by newalway** [comment]
> I found it with class no-display hidden . How did you override this? Thanks in advance.

I override  css class name  ( display:block !important  ) in magento main style sheet and build templete again.

**Comment by newalway** [comment]
> you mean admin order tab?

yes admin tab

**Comment by junnhouse** [comment]
> Hi, I installed the module in magento2.3.2 but the Sp – Order Attachments menu is not coming.Any idea?

your problem is in Store-Configuration menu right? Try to install with composer instead of copy files to app/code

**Comment by Masumisigma** [comment]
Yes, there was a problem with store configuration, Now it is solved Thank you @junnhouse

**Comment by sprankhub** [comment]
#25 should fix it.
